rviz_ogre_vendor/CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ if(WIN32)
 
   ament_vendor(zlib_vendor
     VCS_URL https://github.com/madler/zlib.git
-    VCS_VERSION v1.2.13
+    VCS_VERSION v1.3.1
```

Comment:
Since Ubuntu 24.04 currently has zlib 1.3 (https://packages.ubuntu.com/noble/zlib1g), I'm going to suggest we stick with that for now:

```suggestion
    VCS_VERSION v1.3
```

Reply:
> fixed! to VCS_VERSION v1.3

Reply:
> Since Ubuntu 24.04 currently has zlib 1.3 (https://packages.ubuntu.com/noble/zlib1g), I'm going to suggest we stick with that for now:

fixed